frontend/src/APIClients/CommonAPIClient.ts

```diff
@@ -1,9 +1,15 @@
 import AUTHENTICATED_USER_KEY from "../constants/AuthConstants";
-import { GetLogRecordsReponse } from "../types/LogRecordTypes";
+import {
+  GetLogRecordsReponse,
+  GetLogRecordCountResponse,
+} from "../types/LogRecordTypes";
 import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
 import baseAPIClient from "./BaseAPIClient";
 
-import { LogRecordFilters } from "../components/common/types/Filters";
+import {
+  LogRecordFilters,
+  CountLogRecordFilters,
+} from "../components/common/types/Filters";
 
 const filterLogRecords = async ({
   building = "",
```

Comment:
in the model, building is not nullable, so should we be setting a default of ""?